src/opamBundleMain.ml

```diff
@@ -764,7 +764,13 @@ open Cmdliner
 
 let pkg_version_conv =
   Arg.conv ~docv:"VERSION" (
-    (fun s -> try Ok (OpamPackage.Version.of_string s) with Failure s ->
+    (fun s ->
```

Comment:
`pkg_version_conv` is used for `ocaml` and`opam` versions. It is better to do the conversion in `create_bundle` function (l.88), with a message.

Reply:
Also add a test with unknown opam version